api_app/analyzers_manager/file_analyzers/mobsf_service.py

```diff
@@ -0,0 +1,128 @@
+import logging
+
+import requests
+
+from api_app.analyzers_manager.classes import FileAnalyzer
+from tests.mock_utils import MockUpResponse, if_mock_connections, patch
+
+logger = logging.getLogger(__name__)
+
+
+class MobSF_Service(FileAnalyzer):
+    mobsf_host: str
+    _mobsf_api_key: str
+
```

Comment:
I would add a parameter to enable or not dynamic analysis. Static is the default. Dynamic could be opted out to save time and computation.